src/libhirte/log/log.h

```diff
@@ -0,0 +1,146 @@
+#pragma once
+
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <systemd/sd-journal.h>
+
+#include "libhirte/common/common.h"
+
+typedef enum LogLevel {
+        LOG_LEVEL_DEBUG,
+        LOG_LEVEL_INFO,
+        LOG_LEVEL_WARN,
+        LOG_LEVEL_ERROR,
+} LogLevel;
+
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+static const char *log_level_strings[] = { "DEBUG", "INFO", "WARN", "ERROR" };
```

Comment:
I think maybe we can avoid the lint warning if you go:
```
 static const char * const log_level_strings[] = { "DEBUG", "INFO", "WARN", "ERROR" };
```

Reply:
But also, I don't think you should define this in the header, each .c file that includes this will have its own copy of it. Put it in log.c with a just a declaration of it in the header.

Reply:
Ah ok. I created a `log/log.c` file and moved it there. A declaration of the array is not needed in the header (at least the implementation doesn't require it), only the declaration of the `log_level_to_string` remains there.